chia/wallet/nft_wallet/nft_wallet.py

```diff
@@ -1092,7 +1092,7 @@ async def set_bulk_nft_did(
         for tx in nft_tx_record:
             if tx.spend_bundle is not None:
                 spend_bundles.append(tx.spend_bundle)
-                refined_tx_list.append(dataclasses.replace(tx, spend_bundle=None))
+            refined_tx_list.append(dataclasses.replace(tx, spend_bundle=None))
```

Comment:
why is this part of this PR?

Reply:
It disappeared after merged the main branch.